pom.xml

```diff
@@ -19,94 +19,280 @@
+                            <referenceUrl>
+                                hibernate:ejb3:exo-pu?hibernate.ejb.naming_strategy=org.hibernate.cfg.ImprovedNamingStrategy
+                            </referenceUrl>
+                            <verbose>true</verbose>
+                            <logging>debug</logging>
+                        </configuration>
+                        <dependencies>
+                            <dependency>
+                                <groupId>org.javassist</groupId>
+                                <artifactId>javassist</artifactId>
+                                <version>${org.javassist.version}</version>
+                            </dependency>
+                            <dependency>
+                                <groupId>org.liquibase</groupId>
+                                <artifactId>liquibase-core</artifactId>
+                                <version>${org.liquibase.version}</version>
+                            </dependency>
+                            <dependency>
+                                <groupId>org.liquibase.ext</groupId>
+                                <artifactId>liquibase-hibernate4</artifactId>
+                                <version>${org.liquibase-hibernate4.version}</version>
+                            </dependency>
+                            <dependency>
+                                <groupId>javax.validation</groupId>
+                                <artifactId>validation-api</artifactId>
+                                <version>${validation-api.version}</version>
+                            </dependency>
+                            <dependency>
+                                <groupId>mysql</groupId>
+                                <artifactId>mysql-connector-java</artifactId>
+                                <!-- Replace with the version of the MySQL driver you want to use -->
+                                <version>${mysql.version}</version>
+                            </dependency>
+                            <dependency>
+                                <groupId>org.exoplatform.commons</groupId>
+                                <artifactId>commons-component-common</artifactId>
+                                <version>${org.exoplatform.platform.version}</version>
+                            </dependency>
+                            <dependency>
+                                <groupId>xml-apis</groupId>
+                                <artifactId>xml-apis</artifactId>
+                                <version>${xml-apis.version}</version>
+                            </dependency>
+                        </dependencies>
+                    </plugin>
+                </plugins>
+            </build>
+            <properties>
+                <profile.liquibase>liquibase</profile.liquibase>
+                <db.host>localhost</db.host>
+                <db.port>3306</db.port>
+                <db.schema>tribe</db.schema>
```

Comment:
We should use something less connoted than tribe 😄

Reply:
Will be replaced with «variable d'environnement».